### PR TITLE
Security UI Fixes

### PR DESCRIPTION
**Pull request by jeznorth**
## Description of Changes

- Updating language and front-end clean-up for the security interface.

### CODE REVIEW

File: app/src/components/security/SecurityRuleForm.tsx [hunk 1]
```diff
@@ -224,61 +223,62 @@ const SecurityRuleForm = () => {
...
-          })}
-        </Stack>
-
-        <Box my={2}>
+        <Box component="fieldset" mt={3}>
           <Typography component="legend">Manage Existing Security</Typography>
-          <Typography variant="body2">
-            These rules have already been applied to one or more of the selected features.
+          <Typography variant="body1" color="textSecondary" sx={{ mt: -0.5 }}>
+            Manage security rules have already been applied.
```
Comment (curtisupshall) [inline app/src/components/security/SecurityRuleForm.tsx]: Perhaps a type? "Manage securiy rules **that** have already been applied."

File: app/src/features/submissions/components/SubmissionDataGrid.tsx [hunk 2]
```diff
@@ -84,11 +84,11 @@ export const SubmissionDataGrid = (props: ISubmissionDataGridProps) => {
...
             gap={1}
             sx={{
-              color: 'text.secondary'
+              color: 'error.main'
             }}>
             <Icon path={mdiLockOpenOutline} size={0.75} />
-            <Typography component="span" sx={{ textTransform: 'uppercase' }}>
-              Unsecured
+            <Typography component="span" textTransform="uppercase">
+              Unsecuredasdfasdfsd
```
Comment (curtisupshall) [inline app/src/features/submissions/components/SubmissionDataGrid.tsx]: I think this was unintentionasdfasdfsd

**Review by curtisupshall — DISMISSED** [review]
LGTM

File: app/src/features/submissions/components/SubmissionDataGrid.tsx (hunk 2 above)
Comment (curtisupshall) [reply app/src/features/submissions/components/SubmissionDataGrid.tsx]: Fixed.

File: app/src/components/security/SecurityRuleForm.tsx (hunk 1 above)
Comment (curtisupshall) [reply app/src/components/security/SecurityRuleForm.tsx]: Text removed.

**Review by curtisupshall — DISMISSED** [review]
Resolved comments; Tested locally on my machine. Looks good to me

**Review by al-rosenthal — DISMISSED** [review]
looks good!

**Review by NickPhura — APPROVED** [review]
🔒

**Review by curtisupshall — APPROVED** [review]
🤝

**Pull request merged**